module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -150,6 +157,20 @@ public function setConfig($config)
         $this->tenant = $this->config['API']['tenant'];
     }
 
+    /**
+     * Set main configuration
+     *
+     * Store a reference to the config.ini configuration
+     *
+     * @param array $mainConfig Configuration array loaded from the main VuFind configuration
```

Comment:
I don't think you need both of the lines above, and I'd adjust the `@param` to reference config.ini.

This is obviously not a big deal, but I'm trying to avoid unnecessary references to VuFind® since if we mention it, we should include the ®, and it's less of a hassle to just avoid mentioning the name. ;-)